### DISCUSSION TITLE
Cannot Use Firefox Preferences

### DISCUSSION THREAD

**Discussion by paynetopia** [mozilla/policy-templates#845] [Q&A] [answered]
In Group Policy Management Console in Windows I do see Mozilla >> Firefox >> Preferences (Deprecated); however, I do not see Mozilla >> Firefox >> Preferences. 
![Firefox_Preferences](https://user-images.githubusercontent.com/71395263/146241821-3703f874-8405-46ad-915f-b15635a4956a.png)
Am I not understanding something?

**Comment by mkaply** [comment]
It's in the main list now, not a folder. So just go down farther:

![image](https://user-images.githubusercontent.com/345868/146249735-b67d1a61-4f42-465c-b5a0-e0b2f7d4e315.png)

**Reply by paynetopia** [reply]
I have that setting enabled now, but still confused then on how to set the appropriate settings listed here: https://github.com/mozilla/policy-templates/blob/master/README.md#preferences.

**Reply by paynetopia** [reply]
Wait. Do I enable it and then add the code to the Options? There should be an example in the Help section.

**Comment by mkaply** [comment, marked as answer]
You enable it and then paste the JSON into the options. The link in the help sections shows what to do:

https://github.com/mozilla/policy-templates/blob/master/README.md#preferences

If you scroll down you'll see an example REG_MULTI_SZ

```
{
  "accessibility.force_disabled": {
    "Value": 1,
    "Status": "default"
  },
  "browser.cache.disk.parent_directory": {
    "Value": "SOME_NATIVE_PATH",
    "Status": "user"
  },
  "browser.tabs.warnOnClose": {
    "Value": false,
    "Status": "locked"
  }
}
```

**Reply by paynetopia** [reply]
It was very confusing as it is far different in the way we implement any policy in Windows. Thank you for the assist!

**Reply by mkaply** [reply]
Totally understand. Chrome has been doing some of this as well. Basically as the policy content gets more complicated, it gets really hard to contain it in the structure that Microsoft has created.
